flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -51,6 +56,17 @@
 #endif
 #define MPU6000_SPI_LOW_SPEED               1000000
 
+// Private variables
```

Comment:
will post it in a new  comment at the bottom, so that it is not hidden when these diffs are outdated